rt_gene/scripts/estimate_blink_standalone.py

```diff
@@ -0,0 +1,116 @@
+        if self.viz:
+            #TODO: viz on eye images? cv2.imshow ?
+            viz_img = self.overlay_prediction_over_img(right_eyes[0], blinks[0])
+        return probs, blinks
+
+class BlinkEstimatorVideo(BlinkEstimatorStandalone):
+    # TODO
+    def estimate(self, video_path):
+        pass
+
+
+
+if __name__ == '__main__':
+    parser = argparse.ArgumentParser(description='Estimate blink from images, folder or even sequence.')
+    parser.add_argument('mode', type=str, default='image-pair', help='the input mode', choices=['image-pair', 'folder', 'video'])
+    parser.add_argument('--left', type=str, default=os.path.join(script_path, '../samples/'), nargs='?', help='Path to an image or a directory containing images')
+    parser.add_argument('--right', type=str, default=os.path.join(script_path, '../samples/'), nargs='?', help='Path to an image or a directory containing images')
+    parser.add_argument('--model', nargs='+', type=str, default=[os.path.join(script_path, '../model_nets/Model_allsubjects1.h5')], help='List of blink estimators')
+    parser.add_argument('--threshold', default=0.5, help='List of blink estimators')
+    '''
+    parser.add_argument('input_path', type=str, default=os.path.join(script_path, '../samples/'), nargs='?', help='Path to an image or a directory containing images')
+    parser.add_argument('--calib-file', type=str, dest='calib_file', default=None, help='Camera calibration file')
+    parser.add_argument('--vis-headpose', dest='vis_headpose', action='store_true', help='Display the head pose images')
+    parser.add_argument('--no-vis-headpose', dest='vis_headpose', action='store_false', help='Do not display the head pose images')
+    parser.add_argument('--save-headpose', dest='save_headpose', action='store_true', help='Save the head pose images')
+    parser.add_argument('--no-save-headpose', dest='save_headpose', action='store_false', help='Do not save the head pose images')
+    parser.add_argument('--vis-gaze', dest='vis_gaze', action='store_true', help='Display the gaze images')
+    parser.add_argument('--no-vis-gaze', dest='vis_gaze', action='store_false', help='Do not display the gaze images')
+    parser.add_argument('--save-gaze', dest='save_gaze', action='store_true', help='Save the gaze images')
+    parser.add_argument('--save-estimate', dest='save_estimate', action='store_true', help='Save the predictions in a text file')
+    parser.add_argument('--no-save-gaze', dest='save_gaze', action='store_false', help='Do not save the gaze images')
+    parser.add_argument('--output_path', type=str, default=os.path.join(script_path, '../samples/out'), help='Output directory for head pose and gaze images')
+    parser.add_argument('--models', nargs='+', type=str, default=[os.path.join(script_path, '../model_nets/Model_allsubjects1.h5')],
+                        help='List of blink estimators')
+    '''                    
+    args = parser.parse_args()
+    
+    #TODO arg parser
```

Comment:
If `viz` is always true, remove the parameter and assume it is always true. Or, change the `argparser` to deal with it. Also, remove the TODO line.